Create presenting.md

aldenc opened the pull request:
draft sub-chapter on presenting about TTW

<!--
Please complete the following sections when you submit your pull request. You are encouraged to keep this top level comment box updated as you develop and respond to reviews. Note that text within html comment tags will not be rendered.
-->
### Summary

<!-- Describe the problem you're trying to fix in this pull request. Please reference any related issue and use fixes/close to automatically close them, if pertinent. For example: "Fixes #58", or "Addresses (but does not close) #238". -->

New sub-chapter with guidelines for giving a talk about TTW

### List of changes proposed in this PR (pull-request)

<!-- We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->

* add sub-chapter per https://github.com/alan-turing-institute/the-turing-way/issues/2541


### What should a reviewer concentrate their feedback on?

<!-- This section is particularly useful if you have a pull request that is still in development. You can guide the reviews to focus on the parts that are ready for their comments. We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->

Please review the draft and provide comments on content, style, clarity


### Acknowledging contributors

<!-- Please select the correct box -->

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file: <!-- replace this text with the GitHub IDs of any new contributors -->

sgibson91 commented on book/website/community-handbook/presenting.md:
```diff
@@ -0,0 +1,14 @@
+Whether you are submitting an abstract to a conference or you have been invited to speak, a great thing to do is to engage the Turing Way community for support and to publicize the event. Even if the talk is not open to all, you can work with other community members to refine your slides and give a practice talk if you'd like. Please post the details in Slack under the ______ channel.
+
+### Preparing Slides
+Please create a GitHub issue using the ______ template (link), which provides links to the resources you will need as well as a checklist for preparing your talk. You will find a range of presentation templates, as well as past presentations, in the Promotion Pack folder on Google Drive. Feel free to download the templates and use the slides as a guide for style and content.
```
```suggestion
Please create a GitHub issue using the ["Give a talk" issue template](https://github.com/alan-turing-institute/the-turing-way/issues/new?assignees=&labels=talks-and-workshops%2Cnewsletter&template=give_a_talk.yml&title=%5BTALK%5D+%3Ctitle%3E), which provides links to the resources you will need as well as a checklist for preparing your talk. You will find a range of presentation templates, as well as past presentations, in the Promotion Pack folder on Google Drive. Feel free to download the templates and use the slides as a guide for style and content.
```

Also, do we have a link to the Google Drive folder?

sgibson91 commented on book/website/community-handbook/presenting.md:
```diff
@@ -0,0 +1,14 @@
+We ask that you include certain acknowledgments in your talk to make sure contributors are credited. You'll find all the info you need in the presentation templates as well as the GitHub issue checklist.
+
+### After Your Talk
+Keep spreading the word! Please [save your slides as a PDF](https://www.wikihow.com/Convert-Powerpoint-to-PDF) and upload to [Zenodo](https://zenodo.org/), and don't forget to tag "the-turing-way" under Communities.
```
```suggestion
Keep spreading the word! Please save your slides in their original format and [as a PDF](https://www.wikihow.com/Convert-Powerpoint-to-PDF) and upload to [Zenodo](https://zenodo.org/communities/the-turing-way/), and don't forget to tag "the-turing-way" under Communities.
```

If they upload from this page, the tagging will be done automatically 😉

sgibson91 commented on book/website/community-handbook/presenting.md:
```diff
@@ -0,0 +1,14 @@
+Talks are a great way to engage new communities, teach people about open and reproducible practices, and expand the reach of the Turing Way by introducing the book and inviting new contributors. We encourage any Turing Way contributor to present on behalf of the Turing Way, and we ask that you use the guidelines detailed below to shape your talk and ensure consistency with TTW style. 
+
+### Planning a Talk
+Whether you are submitting an abstract to a conference or you have been invited to speak, a great thing to do is to engage the Turing Way community for support and to publicize the event. Even if the talk is not open to all, you can work with other community members to refine your slides and give a practice talk if you'd like. Please post the details in Slack under the ______ channel.
```
Reminder to fill in the Slack channel name when this has been established

aldenc replied:
@aleesteele did we pick a channel for this, or do we want to create a new one?

EKaroune replied:
we could use the community channel as I think that is the one that most talks and opportunities like that are put in now.

aleesteele replied:
Sorry I'm late to this! Seems like there are three applicable channels here:
#general : usually used to ask questions that aren't 
#community : where community celebrations, news, and organisational-related
#events : used to promote external events (in the wider open science network, etc.)

Would agree that for now, the community channel makes sense. Am currently reviewing the slack channels, so names may change at some point in the future to account for folk's needs: https://docs.google.com/spreadsheets/d/11MmzROZGpSkzoB01ATOTpklXrIg86eKSm1xUmPZuhlA/edit#gid=547657140

aleesteele replied:
Following up on this, I don't think including slack happens naturally within the community. How about using the wording "If you'd like, please post the details in Slack in the #community channel"?

aldenc commented:
@sgibson91 we were considering not linking directly to the google drive so that people can't click there and then forget to use the issue template... what do you think?

sgibson91 commented:
Sorry lost track of this. I think there needs to be _some way_ to get to the google drive if you're going to be referring to it as a source of resources. I'm flexible on what that way is, but we can't hint at some treasure trove of resources and expect people to go looking without a map.

EKaroune commented on book/website/community-handbook/presenting.md:
```diff
@@ -0,0 +1,14 @@
+# Presenting About the Turing Way
+Talks are a great way to engage new communities, teach people about open and reproducible practices, and expand the reach of the Turing Way by introducing the book and inviting new contributors. We encourage any Turing Way contributor to present on behalf of the Turing Way, and we ask that you use the guidelines detailed below to shape your talk and ensure consistency with TTW style. 
```
```suggestion
Talks are a great way to engage new communities, teach people about open and reproducible practices, and expand the reach of _The Turing Way_ by introducing the book and inviting new contributors. We encourage any Turing Way contributor to present on behalf of _The Turing Way_, and we ask that you use the guidelines detailed below to shape your talk and ensure consistency with _The Turing Way_ style. 
```

EKaroune replied:
The Turing Way is the official name and it must be in italics. I would not use TTW as this is unclear to anyone outside of the core team.

EKaroune commented on book/website/community-handbook/presenting.md:
```diff
@@ -0,0 +1,14 @@
+# Presenting About the Turing Way
+Talks are a great way to engage new communities, teach people about open and reproducible practices, and expand the reach of the Turing Way by introducing the book and inviting new contributors. We encourage any Turing Way contributor to present on behalf of the Turing Way, and we ask that you use the guidelines detailed below to shape your talk and ensure consistency with TTW style. 
+
```
Could we add in her a section on: 
### Suggesting venues for talks
If you would like _The Turing Way_ to talk at your event or at a community meeting, please do get in touch with our Community Manager or promote the opportunity on our #community Slack channel.

EKaroune commented on book/website/community-handbook/presenting.md:
```diff
@@ -0,0 +1,14 @@
+We ask that you include certain acknowledgments in your talk to make sure contributors are credited. You'll find all the info you need in the presentation templates as well as the GitHub issue checklist.
+
+### After Your Talk
+Keep spreading the word! Please save your slides in their original format and [as a PDF](https://www.wikihow.com/Convert-Powerpoint-to-PDF) and upload to [Zenodo](https://zenodo.org/communities/the-turing-way/), and don't forget to tag "the-turing-way" under Communities.
```
```suggestion
Keep spreading the word! Please archive your slides in their original format and [as a PDF](https://www.wikihow.com/Convert-Powerpoint-to-PDF) and upload to [Zenodo](https://zenodo.org/communities/the-turing-way/), and don't forget to tag "the-turing-way" under Communities.

Please make sure that you credit all of the contributors and presenters of your presentation as authors in the Zenodo upload form. If possible, link to any previous talks where you have used additional slides outside of the provided _The Turing Way_ presentation templates - this can be done in the description part of the upload form.
```

eirini-zormpa replied:
```suggestion
Keep spreading the word! Please archive your slides in two formats: their original (editable) format and as a PDF. If you don't know how to convert PowerPoint slides to PDF, see these [instructions](https://www.wikihow.com/Convert-Powerpoint-to-PDF); following these instruction helps preserve accessibility features for people using screen readers. You can archive your slides on [Zenodo](https://zenodo.org/communities/the-turing-way/), and don't forget to tag "the-turing-way" under Communities.
```

EKaroune commented on book/website/community-handbook/presenting.md:
```diff
@@ -0,0 +1,14 @@
+# Presenting About the Turing Way
```
```suggestion
# Presenting About _The Turing Way_
```

EKaroune commented on book/website/community-handbook/presenting.md:
```diff
@@ -0,0 +1,14 @@
+Talks are a great way to engage new communities, teach people about open and reproducible practices, and expand the reach of the Turing Way by introducing the book and inviting new contributors. We encourage any Turing Way contributor to present on behalf of the Turing Way, and we ask that you use the guidelines detailed below to shape your talk and ensure consistency with TTW style. 
+
+### Planning a Talk
+Whether you are submitting an abstract to a conference or you have been invited to speak, a great thing to do is to engage the Turing Way community for support and to publicize the event. Even if the talk is not open to all, you can work with other community members to refine your slides and give a practice talk if you'd like. Please post the details in Slack under the ______ channel.
```
```suggestion
Whether you are submitting an abstract to a conference or you have been invited to speak, a great thing to do is to engage _The Turing Way_ community for support and to publicize the event. Even if the talk is not open to all, you can work with other community members to refine your slides and give a practice talk if you'd like. Please post the details in Slack under the ______ channel.
```

EKaroune commented on book/website/community-handbook/presenting.md:
```diff
@@ -0,0 +1,14 @@
+
+### Preparing Slides
+Please create a GitHub issue using the ["Give a talk" issue template](https://github.com/alan-turing-institute/the-turing-way/issues/new?assignees=&labels=talks-and-workshops%2Cnewsletter&template=give_a_talk.yml&title=%5BTALK%5D+%3Ctitle%3E), which provides links to the resources you will need as well as a checklist for preparing your talk. You will find a range of presentation templates, as wel ...
+
```
Add here - You can also see all our previous talks on Zenodo by searching for _The Turing Way_. Here is a [link to _The Turing Way_ Community on Zenodo](https://zenodo.org/communities/the-turing-way/?page=1&size=20).

EKaroune replied:
Are you able to add the link to the promotion pack Google folder?

sgibson91 replied:
An alternative could be to not use Google Drive at all and instead put the promotion pack on zenodo to be downloaded

sgibson91 replied:
> unlike the pdfs uploaded to zenodo.

I take your point RE this specific case. I want to make a general point about the Zenodo. It is a failure point if people are only uploading PDFs to Zenodo. It can handle multiple file formats and people should be sharing **both** in the Zenodo record (PDF + whatever they created the PDF from). Because then the pptx or whatever would also be clonable and editable.

EKaroune reviewed (approved):
Please do ignore my additions if you feel they are not needed apart from the changes to make it _The Turing Way_.

eirini-zormpa reviewed (approved):
Looks good to me! I've reworded the section about converting slides and archiving them to add an explicit mention to accessibility, hope that's okay!

aldenc commented:
@sgibson91 @eirini-zormpa @EKaroune @aleesteele I've added the sub-chapter to the TOC so now you can see it in the preview! I think it's ready to go but would love on final (really, final) review

also not sure how to resolve the conflict in the toc.yml file... when I go to the web editor it shows a whole bunch of lines highlighted rather than just the 2 lines that I added so I'm not sure what's going on

sgibson91 commented:
@aldenc I'm gonna resolve the conflicts for you because it's a complex one

sgibson91 commented on book/website/community-handbook/presenting.md:
```diff
@@ -0,0 +1,16 @@
+We ask that you include certain acknowledgments in your talk to make sure contributors are credited. You'll find all the info you need in the presentation templates as well as the GitHub issue checklist.
+
+### After Your Talk
+Keep spreading the word! Please archive your slides in two formats: their original (editable) format and as a PDF. If you don't know how to convert PowerPoint slides to PDF, see these [instructions](https://www.wikihow.com/Convert-Powerpoint-to-PDF); following these instruction helps preserve accessibility features for people using screen readers. You can archive your slides on [Zenodo](https://z ...
```
Is there a similar guide for exporting Google Slides to PDF? They are probably the 2 big platforms that we should care about being specific about.

aldenc replied:
the wikihow article actually covers both, I will update the text to reflect that
